latest/docs-ref-conceptual/manage-clouds-azure-cli.md

```diff
@@ -15,15 +15,13 @@ ms.devlang: azurecli
 
 # Managing multiple clouds with Azure CLI 2.0
 
-If you have multiple subscriptions associated with Azure, you may have more than one cloud available. Each cloud has its own associated endpoints and capabilities, and is often associated with a particular region that has different data protection standards or requirements.
-
-To effectively work with multiple clouds, you will need to be able to switch between which is currently active, and possibly create new clouds.
+If you work across different regions or organizations with data protection laws in Azure, you may need to use more than one cloud. Each cloud has its own set of available services, restrictions, and data compliance laws for the region or organization it's associated with. This article shows you how to get information on the clouds associated with your active account, change your current cloud, and create or delete new clouds.
```

Comment:
Do we have anything that can define 'cloud' for the user?  The other pages link off to other info about the concepts (tenants, users, etc.)